app/templates/components/validated-input.hbs

```diff
@@ -0,0 +1,24 @@
+{{!-- Copied from ember-cpi-validations/tests/dummy/app - should eventually be part of ember-osf but getting errors with dependencies --}}
```

Comment:
@abought I did change the validated-input.hbs file a bit since there were is a case where a `textarea` had to be validated and not just an `input`. So I just added an extra condition that can be specified when creating the component.